### PR TITLE
fix: handle rpc limit on solana in production environment

### PR DESCRIPTION
**Pull request by tanlethanh**

### CODE REVIEW

File: apps/wallet/browser/kernel/pwa.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { logger } from '@walless/core';
 export const configurePWA = () => {
 	self.addEventListener('install', onInstall);
 	self.addEventListener('activate', onActivate);
-	self.addEventListener('fetch', onFetch);
+	// self.addEventListener('fetch', onFetch);
```
Comment (fluctlight-kayaba) [inline apps/wallet/browser/kernel/pwa.ts:6]: mark as `TODO:` as definitely we'll need to comeback to this

File: apps/wallet/src/engine/runners/solana/collectibles.ts [hunk 2]
```diff
@@ -87,22 +93,17 @@ export const loadCollectibleMetadata = async (
...
-	let nft = await mpl.nfts().load({
-		metadata: metadata as Metadata<JsonMetadata<string>>,
-		tokenOwner: owner,
-	});
-
-	if (!metadata.json && 'mintAddress' in metadata) {
-		const nftByMint = await mpl.nfts().findByMint({
-			mintAddress: metadata.mintAddress,
+	const nft = {
+		...(await mpl.nfts().load({
```
Comment (fluctlight-kayaba) [inline apps/wallet/src/engine/runners/solana/collectibles.ts:97]: I guess we don't need to clone this object, as no one else using/reference it

Comment (tanlethanh) [reply apps/wallet/src/engine/runners/solana/collectibles.ts:97]: actually, the obj return by findByMint is frozen and is read-only

File: apps/wallet/src/engine/runners/solana/history.ts [hunk 3]
```diff
@@ -189,6 +203,7 @@ const getSplTransactionBalances = async (
 	);
 
 	const token: TransactionHistory['token'] = {
+		cluster: '',
```
Comment (fluctlight-kayaba) [inline apps/wallet/src/engine/runners/solana/history.ts:206]: would it be okay to have empty `''` cluster?

**Review by fluctlight-kayaba — APPROVED** [review]

**Pull request merged**